src/Components/Modal.js

```diff
@@ -1,10 +1,8 @@
 import styled from 'styled-components';
-// import img from "./modal.jpg"
-// import { mdiWikipedia } from '@mdi/js';
 import {Icon} from '@iconify/react';
+import Navigation from '../Components/Navigation.js';
 
-export default function Modal({isOpen, onClose, content}) {
-  console.log(content);
+export default function Modal({isOpen, onClose, content, handleClick}) {
```

Comment:
`handleClick` is a very generic name - it would be better to call this prop `onNavigate` or `onChangeElement`. Remember to name functions in props starting with `on` - the handlers you assign to them should be named `handle...`